frontend/src/pages/Details.tsx

```diff
@@ -0,0 +1,38 @@
+function Details () {
```

Comment:
For consistency, can you turn this into a function component i.e. `const Details = () => {}` . Same for all the other pages pls :))